{
	"document_title": "libunbound8-1.13.2-1.2 on GA media",
	"document_type": "SUSE Patch",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "openSUSE-SU-2024:11005-1"
		},
		"status": "Final",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "1",
				"date": "2024-08-23T21:44:06Z",
				"description": "current"
			}
		},
		"initial_release_date": "2024-08-23T21:44:06Z",
		"current_release_date": "2024-08-23T21:44:06Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf.pl",
			"date": "2017-02-24T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "libunbound8-1.13.2-1.2 on GA media",
			"title": "Topic",
			"type": "Summary"
		},
		{
			"text": "These are all security issues fixed in the libunbound8-1.13.2-1.2 package on the GA media of openSUSE Tumbleweed.",
			"title": "Details",
			"type": "General"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		},
		{
			"text": "openSUSE-Tumbleweed-2024-11005",
			"title": "Patchnames",
			"type": "Details"
		}
	],
	"documentdistribution": "Copyright SUSE LLC under the Creative Commons License 4.0 with Attribution (CC-BY-4.0)",
	"document_references": [
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2017-15105/",
			"description": "SUSE CVE CVE-2017-15105 page"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2019-16866/",
			"description": "SUSE CVE CVE-2019-16866 page"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2019-18934/",
			"description": "SUSE CVE CVE-2019-18934 page"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2020-12662/",
			"description": "SUSE CVE CVE-2020-12662 page"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2020-12663/",
			"description": "SUSE CVE CVE-2020-12663 page"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2020-28935/",
			"description": "SUSE CVE CVE-2020-28935 page"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "openSUSE Tumbleweed",
				"branch": [
					{
						"type": "Product Name",
						"name": "openSUSE Tumbleweed",
						"full_product_name": {
							"text": "openSUSE Tumbleweed",
							"product_id": "openSUSE Tumbleweed",
							"cpe": "cpe:/o:opensuse:tumbleweed"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "libunbound8-1.13.2-1.2",
				"full_product_name": {
					"text": "libunbound8-1.13.2-1.2",
					"product_id": "libunbound8-1.13.2-1.2"
				}
			},
			{
				"type": "Product Version",
				"name": "python3-unbound-1.13.2-1.2",
				"full_product_name": {
					"text": "python3-unbound-1.13.2-1.2",
					"product_id": "python3-unbound-1.13.2-1.2"
				}
			},
			{
				"type": "Product Version",
				"name": "unbound-1.13.2-1.2",
				"full_product_name": {
					"text": "unbound-1.13.2-1.2",
					"product_id": "unbound-1.13.2-1.2"
				}
			},
			{
				"type": "Product Version",
				"name": "unbound-anchor-1.13.2-1.2",
				"full_product_name": {
					"text": "unbound-anchor-1.13.2-1.2",
					"product_id": "unbound-anchor-1.13.2-1.2"
				}
			},
			{
				"type": "Product Version",
				"name": "unbound-devel-1.13.2-1.2",
				"full_product_name": {
					"text": "unbound-devel-1.13.2-1.2",
					"product_id": "unbound-devel-1.13.2-1.2"
				}
			},
			{
				"type": "Product Version",
				"name": "unbound-munin-1.13.2-1.2",
				"full_product_name": {
					"text": "unbound-munin-1.13.2-1.2",
					"product_id": "unbound-munin-1.13.2-1.2"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "libunbound8-1.13.2-1.2",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "python3-unbound-1.13.2-1.2",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "unbound-1.13.2-1.2",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "unbound-anchor-1.13.2-1.2",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "unbound-devel-1.13.2-1.2",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "unbound-munin-1.13.2-1.2",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "A flaw was found in the way unbound before 1.6.8 validated wildcard-synthesized NSEC records. An improperly validated wildcard NSEC record could be used to prove the non-existence (NXDOMAIN answer) of an existing wildcard record, or trick unbound into accepting a NODATA proof.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2017-15105",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Tumbleweed:libunbound8-1.13.2-1.2",
						"openSUSE Tumbleweed:python3-unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-anchor-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-devel-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-munin-1.13.2-1.2"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "moderate"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "5",
					"vector_v_2": "AV:N/AC:L/Au:N/C:N/I:P/A:N"
				},
				"score_set_v_3": {
					"base_score_v_3": "5.4",
					"vector_v_3": "CVSS:3.0/AV:N/AC:H/PR:N/UI:N/S:C/C:N/I:L/A:L"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					}
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2017-15105.html",
					"description": "CVE-2017-15105"
				},
				{
					"url": "https://bugzilla.suse.com/1076963",
					"description": "SUSE Bug 1076963"
				}
			]
		},
		{
			"notes": [
				{
					"text": "Unbound before 1.9.4 accesses uninitialized memory, which allows remote attackers to trigger a crash via a crafted NOTIFY query. The source IP address of the query must match an access-control rule.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2019-16866",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Tumbleweed:libunbound8-1.13.2-1.2",
						"openSUSE Tumbleweed:python3-unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-anchor-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-devel-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-munin-1.13.2-1.2"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "important"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "5",
					"vector_v_2": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
				},
				"score_set_v_3": {
					"base_score_v_3": "7.5",
					"vector_v_3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					}
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2019-16866.html",
					"description": "CVE-2019-16866"
				},
				{
					"url": "https://bugzilla.suse.com/1152997",
					"description": "SUSE Bug 1152997"
				}
			]
		},
		{
			"notes": [
				{
					"text": "Unbound 1.6.4 through 1.9.4 contain a vulnerability in the ipsec module that can cause shell code execution after receiving a specially crafted answer. This issue can only be triggered if unbound was compiled with `--enable-ipsecmod` support, and ipsecmod is enabled and used in the configuration.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2019-18934",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Tumbleweed:libunbound8-1.13.2-1.2",
						"openSUSE Tumbleweed:python3-unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-anchor-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-devel-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-munin-1.13.2-1.2"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "important"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "6.8",
					"vector_v_2": "AV:N/AC:M/Au:N/C:P/I:P/A:P"
				},
				"score_set_v_3": {
					"base_score_v_3": "8.1",
					"vector_v_3": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					}
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2019-18934.html",
					"description": "CVE-2019-18934"
				},
				{
					"url": "https://bugzilla.suse.com/1157268",
					"description": "SUSE Bug 1157268"
				}
			]
		},
		{
			"notes": [
				{
					"text": "Unbound before 1.10.1 has Insufficient Control of Network Message Volume, aka an \"NXNSAttack\" issue. This is triggered by random subdomains in the NSDNAME in NS records.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2020-12662",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Tumbleweed:libunbound8-1.13.2-1.2",
						"openSUSE Tumbleweed:python3-unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-anchor-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-devel-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-munin-1.13.2-1.2"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "important"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "5",
					"vector_v_2": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
				},
				"score_set_v_3": {
					"base_score_v_3": "8.6",
					"vector_v_3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:N/I:N/A:H"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					}
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2020-12662.html",
					"description": "CVE-2020-12662"
				},
				{
					"url": "https://bugzilla.suse.com/1171889",
					"description": "SUSE Bug 1171889"
				}
			]
		},
		{
			"notes": [
				{
					"text": "Unbound before 1.10.1 has an infinite loop via malformed DNS answers received from upstream servers.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2020-12663",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Tumbleweed:libunbound8-1.13.2-1.2",
						"openSUSE Tumbleweed:python3-unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-anchor-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-devel-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-munin-1.13.2-1.2"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "important"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "5",
					"vector_v_2": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
				},
				"score_set_v_3": {
					"base_score_v_3": "7.5",
					"vector_v_3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					}
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2020-12663.html",
					"description": "CVE-2020-12663"
				},
				{
					"url": "https://bugzilla.suse.com/1171889",
					"description": "SUSE Bug 1171889"
				}
			]
		},
		{
			"notes": [
				{
					"text": "NLnet Labs Unbound, up to and including version 1.12.0, and NLnet Labs NSD, up to and including version 4.3.3, contain a local vulnerability that would allow for a local symlink attack. When writing the PID file, Unbound and NSD create the file if it is not there, or open an existing file for writing. In case the file was already present, they would follow symlinks if the file happened to be a symlink instead of a regular file. An additional chown of the file would then take place after it was written, making the user Unbound/NSD is supposed to run as the new owner of the file. If an attacker has local access to the user Unbound/NSD runs as, she could create a symlink in place of the PID file pointing to a file that she would like to erase. If then Unbound/NSD is killed and the PID file is not cleared, upon restarting with root privileges, Unbound/NSD will rewrite any file pointed at by the symlink. This is a local vulnerability that could create a Denial of Service of the system Unbound/NSD is running on. It requires an attacker having access to the limited permission user Unbound/NSD runs as and point through the symlink to a critical file on the system.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2020-28935",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Tumbleweed:libunbound8-1.13.2-1.2",
						"openSUSE Tumbleweed:python3-unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-anchor-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-devel-1.13.2-1.2",
						"openSUSE Tumbleweed:unbound-munin-1.13.2-1.2"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "moderate"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "2.1",
					"vector_v_2": "AV:L/AC:L/Au:N/C:N/I:N/A:P"
				},
				"score_set_v_3": {
					"base_score_v_3": "4.4",
					"vector_v_3": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:N/I:H/A:N"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					}
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2020-28935.html",
					"description": "CVE-2020-28935"
				},
				{
					"url": "https://bugzilla.suse.com/1173619",
					"description": "SUSE Bug 1173619"
				},
				{
					"url": "https://bugzilla.suse.com/1179191",
					"description": "SUSE Bug 1179191"
				}
			]
		}
	]
}
